Gordon360/Gordon360.csproj

```diff
@@ -1,815 +1,885 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This seems to be linting changes, not material changes. The actual material changes are in the 10-20 line range, but Git got confused because of the linting (e.g. note that line 2 in the original is split across lines 2-3 in the update. And the following many lines are identical, but all offset by one line.